app/models/access_request.rb

```diff
@@ -11,4 +11,9 @@ def recipient
       email: self.email_address,
     )
   end
+
+  def approve!
+    self.status = 1
```

Comment:
Shouldn't it be the API that sets this?

Reply:
@benilovj let's chat about this because I'm confused but I may just be because the local dev environment is potentially different.